### PR TITLE
Main Menu Scripts

### PR DESCRIPTION
**Pull request by jbe1917**
Added scripts to build a main menu with graphics/sound settings and save function for settings

### CODE REVIEW

File: Assets/Scripts/MainMenu/SoundSettingsManager.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		soundEffectsVolumeSlider.onValueChanged.AddListener(delegate {OnSoundEffectVolumeChange();});
+	}
+
+	public void OnMasterVolumeChange(){
+		soundSettings.masterVolume = AudioListener.volume = masterVolumeSlider.value;
+	}
+
+	public void OnMusicVolumeChange()
+	{
+		//TODO
```
Comment (LickMyPoppy) [inline Assets/Scripts/MainMenu/SoundSettingsManager.cs]: Describe tasks of todo, or if they are unused (default template) remove/comment why

File: Assets/Scripts/MainMenu/SoundSettingsManager.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	}
+
+	public void OnSoundEffectVolumeChange()
+	{
+		//TODO
+	}
+
+	public void SaveSettings()
+	{
+
```
Comment (LickMyPoppy) [inline Assets/Scripts/MainMenu/SoundSettingsManager.cs]: Empty methods

File: Assets/Scripts/MainMenu/SoundSettingsManager.cs (hunk 1 above)
Comment (bitingbits) [inline Assets/Scripts/MainMenu/SoundSettingsManager.cs]: TODO in code

File: Assets/Scripts/MainMenu/SoundSettingsManager.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		soundSettings.masterVolume = AudioListener.volume = masterVolumeSlider.value;
+	}
+
+	public void OnMusicVolumeChange()
+	{
+		//TODO
+	}
+
+	public void OnSoundEffectVolumeChange()
+	{
```
Comment (bitingbits) [inline Assets/Scripts/MainMenu/SoundSettingsManager.cs]: TODO in code

**Pull request merged**